lib/spreewald/frame_steps.rb

```diff
@@ -0,0 +1,77 @@
+# You can append `inside the [name or number] iframe` to any other step.
+# Then the step will operate inside the given iframe.
+# Examples:
+#
+#     Then I should see "Kiwi" inside the 1st iframe
+#     Then I should see "Cherry" inside the fruits iframe
+#     When I press "Save" inside the 2nd iframe
+#
+When /^(.*?) inside the (.*?) iframe$/ do |nested_step, frame_identifier|
+  patiently do
+    frame = find_frame(frame_identifier)
+    page.within_frame(frame) do
+      step nested_step
+    end
+  end
+end.overridable
+
+# nodoc
+When /^(.*?) inside the (.*?) iframe:$/ do |nested_step, frame_identifier, table_or_string|
+  patiently do
+    frame = find_frame(frame_identifier)
+    page.within_frame(frame) do
+      step("#{nested_step}:", table_or_string)
+    end
+  end
+end.overridable
+
+if Gem.loaded_specs['capybara'].version >= Gem::Version.new('3')
```

Comment:
I'm not a 100% sure this is a good way to do the version switch, never seen the use of `Gem.loaded_specs` for that purpose. The usual think I see is `Gem::Version.new(Capybara::VERSION)`. Not sure if this is also good, so you can keep it either way, I guess.

Reply:
I agree. Looking at `Capybara::VERSION` is a better way to do this. Thanks for the suggestion!